corehq/form_processor/migrations/sql_templates/migrate_to_bigint.sql

```diff
@@ -0,0 +1,15 @@
+-- Migrate delta and updated_balance columns to bigint without locking table
+
+-- Create bigint_delta column, copy data from delta column, and then rename
+ALTER TABLE form_processor_ledgertransaction ADD COLUMN bigint_delta BIGINT;
+UPDATE form_processor_ledgertransaction SET bigint_delta = delta;
+ALTER TABLE form_processor_ledgertransaction ALTER COLUMN bigint_delta SET NOT NULL;
+ALTER TABLE form_processor_ledgertransaction DROP COLUMN delta;
+ALTER TABLE form_processor_ledgertransaction RENAME COLUMN bigint_delta TO delta;
+
+-- Create bigint_updated_balance column, copy data from updated_balance column, and then rename
+ALTER TABLE form_processor_ledgertransaction ADD COLUMN bigint_updated_balance BIGINT;
+UPDATE form_processor_ledgertransaction SET bigint_updated_balance = updated_balance;
+ALTER TABLE form_processor_ledgertransaction ALTER COLUMN bigint_updated_balance SET NOT NULL;
+ALTER TABLE form_processor_ledgertransaction DROP COLUMN updated_balance;
+ALTER TABLE form_processor_ledgertransaction RENAME COLUMN bigint_updated_balance TO updated_balance;
```

Comment:
Do you know how long each of these two blocks will take on our larger environments? If it will take a long time (more than a few seconds) I suggest breaking them up into two separate migrations so they are executed in separate transactions.

Reply:
Yeah. After testing locally I was going to do that. ... 

What's more, it can break with this:
```
    django.db.utils.OperationalError: cannot ALTER TABLE "form_processor_ledgertransaction" because it has pending trigger events
```